### PR TITLE
add easyconfig for MATLAB 2010b + deps

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/m/MATLAB/MATLAB-2010b.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+version = '2010b'
+
+homepage = 'http://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment 
+that enables you to perform computationally intensive tasks faster than with 
+traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%s-%s.tar.gz'%(name.lower(), version)]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/MATLAB/MATLAB-2010b.eb]: for the record, a pure pythonic .iso extraction appears possible via: http://unix.stackexchange.com/a/164505

Comment (JensTimmerman) [inline easybuild/easyconfigs/m/MATLAB/MATLAB-2010b.eb]: it's not pythonic at al,`  It's a bit hacky, but it gets the job done.`

**Comment by rjeschmi** [comment]
Maybe this can be closed? There are a number of other MATLAB PRs and some of them have been merged even

https://github.com/hpcugent/easybuild-easyconfigs/pull/1918

**Comment by boegel** [comment]
@rjeschmi: agreed, the added value of this is marginal with #1918 merged in

**Pull request closed without merging**